src/analysis/seminorm.lean

```diff
@@ -593,6 +618,17 @@ end
 lemma symmetric_ball_zero (r : ℝ) (hx : x ∈ ball p 0 r) : -x ∈ ball p 0 r :=
 balanced_ball_zero p r (-1) (by rw [norm_neg, norm_one]) ⟨x, hx, by rw [neg_smul, one_smul]⟩
 
+@[simp]
+lemma preimage_neg_ball (p : seminorm 𝕜 E) (r : ℝ) (x : E) :
+  has_neg.neg ⁻¹' ball p x r = ball p (-x) r :=
```

Comment:
```suggestion
lemma neg_ball (p : seminorm 𝕜 E) (r : ℝ) (x : E) :
  -ball p x r = ball p (-x) r :=
```
and I would maybe make it go the other way, in the style of `neg_smul` and `smul_neg`.